richie/js/components/searchFilterGroupContainer/searchFilterGroupContainer.tsx

```diff
@@ -15,16 +15,16 @@ const hardcodedFilters: { [machineName: string]: FilterDefinition; } = {
     humanName: 'Language',
```

Comment:
have you thought of i18n?

Reply:
There's a Trello card for it (https://trello.com/c/ax6JbVfZ/74-manage-i18n-in-ts-tsx-files) but I think now that the project is public it's better to create issues so here it is: https://github.com/openfun/richie/issues/89

Right now I think it's better to move the features forward as fast as we can but it will soon be time to take care of this. I have already done some rough work on this.